src/main/io/gps.c

```diff
@@ -437,6 +557,9 @@ void gpsInitUblox(void)
                 if (lookupBaudRateIndex(serialGetBaudRate(gpsPort)) != newBaudRateIndex) {
                     // change the rate if needed and wait a little
                     serialSetBaudRate(gpsPort, baudRates[newBaudRateIndex]);
+#if 0
```

Comment:
Is this debug still of potential use? If so then define a new debug mode and use `DEBUG_SET`. If this was only needed for initial debug to develop the code, but is of no further use, then it would be best to remove it. Same for similar cases below.

Reply:
It is of potential use, but it's not worth defining a new debug mode, as there are too many variables potentially interesting to be tracked. It's just that I find myself spilling debug[x]'s over the code every time, and since there was an existing #if 0, I thought it was acceptable to keep those around for development purposes.

Reply:
@TonyBlit we are low on flash space. Please remove unneeded code so we can move forward with merging this,

Reply:
done, although this has not saved flashspace (as expected), fyi @haslinghuis 
if you confirm that is all, I'll squash

Reply:
Could you at least change the `#if 0` for `#ifdef DEBUG_XXX` to make it clearer what the debug is being used for as a development aid. Thanks.